geoh5py/shared/concatenation/concatenator.py

```diff
@@ -488,8 +495,12 @@ def remove_entity(self, entity: Concatenated):
                 object_ids.remove(as_str_if_uuid(entity.uid).encode())
                 self.concatenated_object_ids = object_ids
 
-        if self.concatenated_attributes is not None:
+        if (
```

Comment:
It 's not for this review but I'm more and more sceptic about those "

def __init__(self):
   self._attr: mytype | None = None
   
and 

@property
def attr(self):
   if self._attr is None:
      # do something
    else return self._attr
    
It feels like it's not updating, introducing unecessary types management.

Reply:
That pattern can be nice to just force update by deleting the _attr. But I agree, can be dangerous. It's a case by case I guess.

Reply:
Then an updating function?
I don't know, maybe that's something to have to our "good practice" and decide once and for all* 

*can of course change in time ;-)